layer/CMakeLists.txt

```diff
@@ -47,10 +48,9 @@ endif()
 add_definitions(-DVK_ENABLE_BETA_EXTENSIONS)
 
 if (WIN32)
-    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_CRT_SECURE_NO_WARNINGS /bigobj")
-    set (CMAKE_C_FLAGS   "${CMAKE_C_FLAGS} -D_CRT_SECURE_NO_WARNINGS /bigobj")
+    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /bigobj")
+    set (CMAKE_C_FLAGS   "${CMAKE_C_FLAGS} /bigobj")
 else()
-    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wpointer-arith")
     set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wpointer-arith")
```

Comment:
@juan-lunarg  Not sure how to handle CMAKE_C_FLAGS , considering that `pointer-arith` is in `lunarg_target_compiler_configurations`

Reply:
Ok, I am revoming it then!